sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/BlobPartitionManager/BlobPartitionManagerLiveTests.cs

```diff
@@ -178,6 +217,50 @@ await using (StorageScope storageScope = await StorageScope.CreateAsync())
             }
         }
 
+        /// <summary>
+        ///    Verifies that returned eTag contains single quotes
+        /// </summary>
+        ///
+        [Test]
+        public async Task CheckReturnedEtagContainsSingleQuotes()
+        {
+            await using (StorageScope storageScope = await StorageScope.CreateAsync())
+            {
+                // A regular expression used to capture strings enclosed in double quotes.
+                Regex s_doubleQuotesExpression = new Regex("\"(.*)\"", RegexOptions.Compiled);
+
+                var storageConnectionString = StorageTestEnvironment.StorageConnectionString;
+                var containerClient = new BlobContainerClient(storageConnectionString, storageScope.ContainerName);
+
+                var partitionManager = new BlobPartitionManager(containerClient);
+                var ownershipList = new List<PartitionOwnership>();
+                var ownership =
+                    new MockPartitionOwnership
+                    (
+                        "namespace",
+                        "eventHubName",
+                        "consumerGroup",
+                        "ownerIdentifier",
+                        "partitionId"
+                    );
+
+                ownershipList.Add(ownership);
+
+                IEnumerable<PartitionOwnership> claimedOwnership = await partitionManager.ClaimOwnershipAsync(ownershipList);
+
+                IEnumerable<PartitionOwnership> storedOwnershipList = await partitionManager.ListOwnershipAsync("namespace", "eventHubName", "consumerGroup");
+
+                Match claimedOwnershipMatch = s_doubleQuotesExpression.Match(claimedOwnership.First().ETag);
+                Match storedOwnershipListMatch = s_doubleQuotesExpression.Match(storedOwnershipList.First().ETag);
+
+                Assert.That(storedOwnershipList, Is.Not.Null);
+                Assert.That(storedOwnershipList.Count, Is.EqualTo(1));
+                Assert.That(claimedOwnershipMatch.Success, Is.False);
+                Assert.That(storedOwnershipListMatch.Success, Is.False);
```

Comment:
I think you can keep just these. Not sure why we are doing the other three assertions.